tendrl/node_agent/manager/manager.py

```diff
@@ -227,6 +228,34 @@ def on_pull(self, raw_data):
                     node_id=raw_data["node_id"],
                 )
             )
+        if "disks" in raw_data:
+            try:
+                self.etcd_client.delete(
+                    ("nodes/%s/Disks") % raw_data["node_id"], recursive=True)
+            except Exception as ex:
```

Comment:
Dont catch generic exception and simply log it. It can be any sort of exception, find python-etcd specific exception and catch that https://github.com/jplana/python-etcd/blob/master/src/etcd/__init__.py#L118...L307